lib/Sema/CSBindings.cpp

```diff
@@ -874,14 +874,12 @@ bool LiteralRequirement::isCoveredBy(Type type, DeclContext *useDC) const {
   if (hasDefaultType() && coversDefaultType(type, getDefaultType()))
     return true;
 
-  return (bool)TypeChecker::conformsToProtocol(type, getProtocol(),
```

Comment:
conformsToProtocol() also checks conditional requirements; is that an issue here?

Reply:
It's actually better if it doesn't check them here, leads to better diagnostics.